src/app/account/mfa/panel.tsx

```diff
@@ -46,12 +46,10 @@ export function ManageMFAPanel() {
                     <Text size="md" mb="md">
                         Feel free to opt in to use either SMS verification OR Authenticator App verification.
                     </Text>
-                    <Link href="/account/mfa/sms">
-                        <Button>SMS Verification</Button>
-                    </Link>
-                    <Link href="/account/mfa/app">
-                        <Button>Authenticator App Verification</Button>
-                    </Link>
+                    <Flex gap="md">
+                        <ButtonLink href="/account/mfa/sms" styles={{ root: { margin: 1 } }}>SMS Verification</ButtonLink>
```

Comment:
```styles={{ root: { margin: 1 } }}```
this scares me 😨

Reply:
yes, I see that ButtonLink supports all the props that Button takes, so pretty sure you can use mantine style props: https://mantine.dev/styles/style-props/ on it.  try something like this:
```suggestion
                        <ButtonLink href="/account/mfa/sms" m="xs">SMS Verification</ButtonLink>
```

that won't quite be a 1px, but pretty close.  we much prefer to use those over raw styles.

Reply:
although, why are we setting the margin on the buttons at all? shouldn't the gap on the flex that you added handle that? I'm guessing I'm mis-understanding what this is intended to do, sorry.

Reply:
sorry :( what do you normally do?

Reply:
the `:focus` border is touching the button if the margin isn't set. adding a margin makes it more visible

Reply:
the mantine props worked out! thanks nathan and chris